python/lsst/obs/decam/rawFormatter.py

```diff
@@ -0,0 +1,39 @@
+# This file is part of obs_decam.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Gen3 Butler Formatters for Dark Energy Camera raw data.
+"""
```

Comment:
Given a choice between a module docstring and a class docstring with exactly the same information content, I'd pick the class docstring.  But I'm also curious to get @jonathansick 's thoughts on whether we should have both (and which we should prefer) when they'd say essentially the same thing, as this is something that comes up pretty often.  I do think that the module docstring should probably be left out if all of the public symbols are lifted to package scope and are expected to be used there, but that's probably not actually the case with this particular file.

Reply:
Good point. In this case, should we not supply any class-level docstring, and just defer to the parent `Instrument`?

Reply:
I think it's good practice to document subclasses at the class level even if you don't document any of their methods.